content/blog/jt15s/2020-12-31_hello_from_the_haiku_promotion_team.md

```diff
@@ -0,0 +1,56 @@
+* Graphic Designer - it would be nice if we had someone who was solely responsible for producing graphics for Haiku promotion. Graphics should be uploaded to both the website and Haiku Artwork repository using Github, but I’m more than happy to help teach people how to do that.
+* Live Stream Technician - we are considering doing live streams, but we will need someone with technical knowledge of live-streaming who can help facilitate this.
+* Events Coordinator - it would be good to have someone search for events which Haiku could take part in (i.e. FOSS conferences) and help organise Haiku’s participation in these events.
+
+## Our promotional strategy
+As the promotion team, we want to make sure people know about Haiku. One of the biggest problems facing the project is that we are more obscure than other FOSS projects. This means it's harder to get new contributors, whether this is developers, documentation writers or just volunteers in general. Hence we aim to attract more attention towards Haiku, and in turn, new users and volunteers. 
+
+**I would like to take this opportunity to urge you to help the project: you don't need to be a programmer to help out. [You can find ways to get involved here.](https://www.haiku-os.org/community/getting-involved/)**
+
+How will we do this, you might ask? Well, there are a couple of aspects of our strategy. 
+
+### The media and reviewers
+A database of all the reviews/mentions of Haiku is being compiled for easy reference and so we can contact a variety of media sources when we have a new release. Additionally, we are looking for more technology-related channels who may have not mentioned or reviewed Haiku before and see if they are interested in having a closer look at the OS. 
+
+In terms of the media in general, we are planning to prepare press releases for the media - a press release is simply a formal, properly formatted version of an announcement. By releasing press releases it makes it easier for other media organisations to find major Haiku announcements and run a story about Haiku. 
+
+In terms of reviewers, as mentioned before we are planning to contact reviewers and media organisations once Beta 3 is released. We have also prepared a reviewer information sheet with tips for reviewers, which is in the draft stage. 
+
+### Businesses
+Businesses can benefit from Haiku as an operating system, and personally, I see Haiku as a lightweight alternative to the proprietary desktop operating systems businesses currently use. 
+
+An information sheet for businesses has been drafted, which gives businesses a rundown of Haiku and its advantages and disadvantages as a business-oriented operating system. However, to attract more businesses we must improve Haiku further, and hence we will delay marketing efforts for businesses until Haiku reaches a stable and mature stage. 
+
```

Comment:
It's a bit strange to me that this strategy mentions developers and businesses, but skips over the main focus of Haiku (in it's finalized form): regular users, running them at home on their personal computers.

While we're not quite ready for a general release (my personal metric is "would I install Haiku on my parent's computer?"), we already have some early adopters users. People who appreciate Haiku for what it is and what it tries to be, and are ready to live with the limitations. This is also something we can build upon, even if it needs some care to not set too high expectations.

Reply:
@pulkomandy Agreed - thanks for pointing that out! I will write a paragraph mentioning general users as soon as I can.